src/components/teaser-paid/teaser-paid.scss

```diff
@@ -0,0 +1,33 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  // Use !important here to not interfere with Firefox focus ring definition
+  // which appears in normalize css of several frameworks.
+  outline: none !important;
+
+  @include sbb.panel-variables;
```

Comment:
teaser paid should not have relations to panel. min-height can probably be removed
```suggestion
```